src/main/java/eu/bbmri_eric/negotiator/common/exceptions/NegotiatorExceptionHandler.java

```diff
@@ -48,6 +54,35 @@ public final ResponseEntity<HttpErrorResponseModel> handleJwtDecoderError(
     return new ResponseEntity<>(errorResponse, HttpStatus.INTERNAL_SERVER_ERROR);
   }
 
+  @ExceptionHandler(MaxUploadSizeExceededException.class)
+  @ApiResponse(
+      responseCode = "413",
+      description = "Payload Too Large - Max file size exceeded",
```

Comment:
Could we add the max allowed file size here, e.g. "Max file size of XXMB exceeded".

Reply:
it is sent in the actual error message, this is just an example that is shown in swagger.
![Screenshot 2024-11-21 at 16 12 11](https://github.com/user-attachments/assets/72159243-bfcc-4fb7-9e75-ca55b27247f3)